tests/core/test_broker.py

```diff
@@ -310,6 +311,23 @@ async def test_no_pika_logs_if_no_debug_mode(caplog: LogCaptureFixture):
         for record in caplog.records
     )
 
+    broker = PikaEventBroker(
+        "host", "username", "password", retry_delay_in_seconds=1, connection_attempts=1
+    )
+
+
+def test_raise_exception_port(monkeypatch: MonkeyPatch):
```

Comment:
If there is no need for MonkeyPatch, this argument can be removed.

Reply:
yes.. sorry, fixed!!